### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by NeverovDS**
Correct version. Sorry, miss click.

### CODE REVIEW

File: kvadrat.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	start    Point
+	a        int
+	End      PointEnd
+	Perimeter int
+	Area     int
+}
+type Point struct {
+	x, y int
+}
+type PointEnd struct {
```
Comment (imatsiash-ib) [inline kvadrat.go]: Why do you need a separate structure for the `End` point ?

File: kvadrat.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+)
+
+type Square struct {
```
Comment (imatsiash-ib) [inline kvadrat.go]: `Square` structure should contain only `Start`and `a` fields (see lecture please)

File: kvadrat.go (hunk 1 above)
Comment (NeverovDS) [reply kvadrat.go]: frankly, it was more convenient

File: kvadrat.go (hunk 2 above)
Comment (NeverovDS) [reply kvadrat.go]: in which structure to put the fields: End,Perimeter,Area?

File: kvadrat.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package main
```
Comment (imatsiash-ib) [inline kvadrat.go]: Please remove incorrect version of solution.

File: kvadratV2.go [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+func (NewPointX *Square) End() Point {
+	x2 := NewPointX.start.x + NewPointX.a
+	y2 := NewPointX.start.y - NewPointX.a
+	return Point{x2, y2}
+}
+func (PerimeterDo *Square) Perimeter() int {
+	return PerimeterDo.a * 4
+}
+func (AreaDo *Square) Area() int {
```
Comment (imatsiash-ib) [inline kvadratV2.go]: Please use the same name for receiver. It is a good practice to use short name (1 or 2 letters).

**Comment by imatsiash-ib** [comment]
Please use different names for PRs.

**Pull request closed without merging**